widget/net.lua

```diff
@@ -9,28 +9,30 @@
 local helpers = require("lain.helpers")
 local naughty = require("naughty")
 local wibox   = require("wibox")
+local os      = { execute = os.execute }
 local string  = { format = string.format, match = string.match }
 
 -- Network infos
 -- lain.widget.net
 
 local function factory(args)
-    local net      = { widget = wibox.widget.textbox(), devices = {} }
-    local args     = args or {}
-    local timeout  = args.timeout or 2
-    local units    = args.units or 1024 -- KB
-    local notify   = args.notify or "on"
-    local screen   = args.screen or 1
-    local settings = args.settings or function() end
+    local net       = { widget = wibox.widget.textbox(), devices = {} }
+    local args      = args or {}
+    local timeout   = args.timeout or 2
+    local units     = args.units or 1024 -- KB
+    local notify    = args.notify or "on"
+    local wifi_info = args.wifi_info or "off"
+    local eth_info  = args.eth_info or "off"
```

Comment:
The variables `{wifi,eth}_info` needs to be documented in the wiki. The notes you wrote in this commit description should also be reported.